### PR TITLE
Create 2014-08-04-spree-girls-blog-post.md

### PR DESCRIPTION
**Pull request by tracymu**
Adding Tracy Mu Sung and Catherine Jones' blog post for Spree Girls

### CODE REVIEW

**Comment by carpodaster** [comment]
Thanks for the PR! Would you mind adding a picture or two? :camera: :smile:

File: blog/_posts/2014-08-04-spree-girls-blog-post.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+The Spree Girls are Tracy Mu Sung and Catherine Jones, we are working on Spree eCommerce and live in Sydney Australia. There are 2 teams working on Spree this year, and we think this is fine since it is such a massive project.
+
+We both work full time, so are undertaking the RGSoC part time outside of our jobs. We meet up one night during the week, all day Sunday, and work remotely from each other the rest of the time. We make good use of Skype and email during the week, and have Google Hangouts with our mentor Trung Le in Melbourne (although we hate Google Hangouts).
+
+###How We Got Here
+
+Catherine is a former fashion designer from the UK, who went back to college to study multimedia design. Once exposed to code she became so enamoured that she went on to study web design. She was taught using C# and PHP, but quickly found out how much better Ruby was.
+
+Tracy became exposed to coding while working in Digital Marketing. She met Catherine at one of the many Sydney Ruby events, and both have become actively involved in the community, attending Ruby Conf, Rails Camps, Hack Nights and ending up being mentors at a Rails Girls.
```
Comment (carpodaster) [inline blog/_posts/2014-08-04-spree-girls-blog-post.md]: > … being mentors at a Rails Girls.

at a Rails Girls workshop?

File: blog/_posts/2014-08-04-spree-girls-blog-post.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+permalink: blog/spree-girls-downunder
+current: blog
+author: Spree Girls
+---
+
+#Spree Girls Australia
+
+The Spree Girls are Tracy Mu Sung and Catherine Jones, we are working on Spree eCommerce and live in Sydney Australia. There are 2 teams working on Spree this year, and we think this is fine since it is such a massive project.
+
+We both work full time, so are undertaking the RGSoC part time outside of our jobs. We meet up one night during the week, all day Sunday, and work remotely from each other the rest of the time. We make good use of Skype and email during the week, and have Google Hangouts with our mentor Trung Le in Melbourne (although we hate Google Hangouts).
```
Comment (carpodaster) [inline blog/_posts/2014-08-04-spree-girls-blog-post.md:14]: Maybe explicitely state the term »voluntary team« and link to http://railsgirlssummerofcode.org/blog/everything-you-ever-wanted-to-know-about-volunteer-teams/ and/or http://railsgirlssummerofcode.org/blog/final-team-list-2014/#volunteering-teams ?

**Comment by tracymu** [comment]
Ok I added some pictures and linked to the volunteer team stuff.

Cheers

**Comment by carpodaster** [comment]
Awesome! Thanks for the blog post :smile:  

We've just published the one from Team Species+ and will hit the merge button tomorrow or on thursday!

**Comment by anikalindtner** [comment]
awesome post, ladies! <3

awesomegirlbiker

 we can publish it today... maybe @FloorD or @alicetragedy can take care of tweeting about it?

**Comment by alicetragedy** [comment]
:thumbsup: it will be done, @anikalindtner! :tada:

**Comment by anikalindtner** [comment]
WHOOP! thank you <3

**Pull request merged**